upup/pkg/fi/cloudup/openstack/instance.go

```diff
@@ -46,7 +47,8 @@ func (c *openstackCloud) CreateInstance(opt servers.CreateOptsBuilder) (*servers
 }
 
 func (c *openstackCloud) DeleteInstance(i *cloudinstances.CloudInstanceGroupMember) error {
-	return fmt.Errorf("openstackCloud::DeleteInstance not implemented")
+	glog.Warning("This does not work without running kops update cluster --yes in another terminal")
```

Comment:
Surely this can be done without invoking twice.  I think we can move forward with this for now.